src/colmap/sfm/incremental_triangulator.cc

```diff
@@ -753,4 +753,12 @@ bool IncrementalTriangulator::HasCameraBogusParams(const Options& options,
   }
 }
 
+std::ostream& operator<<(std::ostream& stream,
+                         const IncrementalTriangulator& triangulator) {
+  stream << "IncrementalTriangulator(reconstruction="
+         << triangulator.reconstruction_ << ", correspondence_graph="
+         << *THROW_CHECK_NOTNULL(triangulator.correspondence_graph_) << ")";
```

Comment:
Make robust to missing correspondence graph.

Reply:
Add test.

Reply:
Did you add the test somewhere for this? I did not find it.

Reply:
No, this was a note for myself here :-) Done now.